geometry/convex_hull.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include <vector>
+
+namespace m2
+{
+// Builds a convex hull around |points|.  The hull polygon points are
+// listed in the order of a counterclockwise traversal.  The hull
+// polygon does not have three points lying on the same straight line.
```

Comment:
The hull polygon points are listed in the order of a counterclockwise traversal with no three points lying on the same straight line.

(I repeated the same prefix but aligned the sentences vertically. With automatic format it's better to avoid repetition.)